src/components/ProductTable.jsx

```diff
@@ -10,15 +10,31 @@ const PRODUCTS = [
   { name: 'Mango', price: '$2.00', stocked: false }
 ];
 
-const ProductTable = () => {
-  // useState hook replaces need for call to setState()
-  let [inStockOnly, setInStockOnly] = useState(false);
-  
+const ProductTable = (props) => {
+  // QUESTION: is it best practice to initialize state within a child component, if that child component is the only place it will be used (i.e. other components don't need to know about it/its current value?)
+  // or is it best practice to initialize state as high up the hierarchy/as globally as possible (i.e. ProductPage) and pass it down as far as it needs to go (in this case, just down one level to this file)?
+  // second option seems to defeat the purpose of creating ProductPage in the first place (other than to handle count state)?
+  // let [inStockOnly, setInStockOnly] = useState(false);
+  console.log('product table props: ', props);
+
   const allProducts = PRODUCTS.map((product) => {
-    return <Product key={product.name} name={product.name} price={product.price} stocked={product.stocked}/>
+    return <Product 
+      key={product.name}
+      name={product.name}
+      price={product.price}
+      stocked={product.stocked}
+      count={props.count}
+      setCount={props.setCount}
+      onIncrement={props.onIncrement}
+      onDecrement={props.onDecrement}
+      // QUESTION: ditto from similar question on ProductPage (line 38)
+      // onIncrement={props.onIncrement(productCount)}
```

Comment:
The function can be passed down and used by the child component but I wouldn't pass the count back as an arg on the function unless there is something that the component knows that the parent wouldn't know. 
(i.e. the clicked item's id or something like that). In this case, we know the count in the parent and don't need to pass it back